CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # UNRELEASED
```

Comment:
Let's remove this.  You can add it in when creating the PR to update the version to snapshot in Phase III: https://github.ibm.com/cloudant/integrations/blob/master/releasing/README.md#update-the-version-number

Reply:
Yeah this line needs to go. I think it will break the automated release stuff.

Reply:
Done in 3f4ae5d7c3dc79a5b85a2b481b998570c262b0fd